python/lsst/analysis/tools/tasks/base.py

```diff
@@ -31,6 +31,8 @@
 connection classes and should specify a unique name
 """
 
+__all__ = ("AnalysisBaseConfig", "AnalysisPipelineTask")
```

Comment:
Is `AnalysisBaseConnections` needed as well?  See diaObjectTableAssocAnalysis.py and others.

Reply:
I've generally assumed that Connections classes should not be part of the public API, since you never need to import them as far as I'm aware. Configs do get imported and modified. I've asked on #dm-science-pipelines about this.